README.md

```diff
@@ -31,25 +31,21 @@ Run `yarn start`
 
 Run `yarn test`
 
-<!-- TODO: rewrite outdated SystemD scripts and update docs.
 ## Installing the development server SystemD Unit File
 
-As a temporary provision until a build script is ready for Tegh the server can be installed with systemd via the following steps:
+As a temporary provision until a build script is available for Tegh, the server can be installed with systemd via the following steps:
 
-1. Run:
+1. Symlink the Tegh server:
   ```
-    sudo ln -s `pwd`/packages/tegh-host-posix/scripts/tegh-host-posix /usr/sbin/tegh-host-posix
-    sudo cp `pwd`/packages/tegh-host-posix/scripts/tegh-host-posix.service /etc/systemd/system/
+    sudo ln --symbolic --target-directory=/usr/local/bin `pwd`/packages/tegh-host-posix/scripts/tegh-server
```

Comment:
Use bin instead of sbin since server daemon is run as the user and not a system or root user.
Use the prefix /usr/local instead of /usr as the location for host-/machine-specific files.